### PR TITLE
chore: try to make audit happy

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

- patch sentry
- bump rustls, ureq, reqwest, etc
- mysql_async disable rustls, enable native-tls

- Closes #issue

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/12562)

### CODE REVIEW

File: tests/sqllogictests/Cargo.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ common-exception = { path = "../../src/common/exception" }
 env_logger = "0.10.0"
 futures-util = "0.3.25"
 lazy_static = { workspace = true }
-mysql_async = { version = "0.31", default-features = false, features = ["rustls-tls"] }
+mysql_async = { version = "0.31", default-features = false, features = ["native-tls-tls"] }
```
Comment (Xuanwo) [inline tests/sqllogictests/Cargo.toml:23]: It's better not to use native-tls.

Comment (PsiACE) [reply tests/sqllogictests/Cargo.toml:23]: It's just for testing purposes, I think it's acceptable, or I can write a patch to remove webpki.

Comment (Xuanwo) [reply tests/sqllogictests/Cargo.toml:23]: > It's just for testing purposes, I think it's acceptable

Query Service also changed which means our binary will need openssl. I prefer to ignore this audit warning.

Comment (PsiACE) [reply tests/sqllogictests/Cargo.toml:23]: > Query Service also changed which means our binary will need openssl. I prefer to ignore this audit warning.

That's too bad, I'll write a patch.

Comment (Xuanwo) [reply tests/sqllogictests/Cargo.toml:23]: Maybe we can fork webpki to https://github.com/tent-rs?

Comment (PsiACE) [reply tests/sqllogictests/Cargo.toml:23]: > Maybe we can fork webpki to https://github.com/tent-rs?

The approach upstream is to use webpki-roots, as webpki has been completely abandoned. I just need to remove it from mysql_async.

Comment (Xuanwo) [reply tests/sqllogictests/Cargo.toml:23]: > The approach upstream is to use webpki-roots, as webpki has been completely abandoned. I just need to remove it from mysql_async.

Makes sense

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**